gatsby-config.js

```diff
@@ -26,13 +26,6 @@ module.exports = {
         ],
       },
     },
-    {
-      resolve: 'gatsby-plugin-i18n',
```

Comment:
Entfernung des `gatsby-plugin-i18n`-Plugins, da nicht mehr benötigt. Wurde genutzt, um die Übersichtsseite in deutsch und englisch zu lokalisieren. Diese werden nun dynamisch zur Build-Phase auf Basis eines Templates und eines erzeugt.